docs/visual-basic/language-reference/operators/operator-precedence.md

```diff
@@ -34,7 +34,7 @@ When several operations occur in an expression, each part is evaluated and resol
 
 ## Precedence Order
 
- Operators are evaluated in the following order of precedence:
+ Operators are evaluated in the following order of precedence. Operators listed first have higher precedence and are evaluated before operators listed later:
```

Comment:
It's not clear that the "following" refers to the H3 headings order, so I'd add an index to the H3 headings here.